INSTALL.md

```diff
@@ -119,12 +118,28 @@ sudo modprobe dattobd
 
 If you would like to have the module be loaded automatically during boot, consult the documentation for your distribution.
 
-### Troubleshooting
+## Installation Troubleshooting
 On some systems, it may be necessary to let the system know of the location of the shared libraries. If you are having trouble getting `dbdctl` to run, run these two commands:
 ```
 echo /usr/local/{lib,lib64} | sed 's/ /\n/g' | sudo tee /etc/ld.so.conf.d/dattobd.conf
 sudo ldconfig
 ```
 
-### Usage
+If you are getting errors that may indicated that the module is not installed, you can ensure the module installed correctly by running the following commands. 
+
+This should return no output if the module was installed:
+```
+modprobe dattobd 
+```
```

Comment:
This command is already in "Configuring the Kernel Module". It would be better to merge this information with that.